Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -43,59 +43,83 @@ You can simply select _DefaultMixedRealityToolkitConfigurationProfile_ and then
-    - At the bottom of the _'PointerProfile'_, you should assign an invisible cursor prefab to the _'GazeCursor'_. If you downloaded the MRTK Examples folder, you can simply reference the included -'EyeGazeCursor'_ prefab.
+    - Navigate to your MRTK configuration profile -> _'Input'_ -> _'Pointers'_
+    - Clone the _'DefaultMixedRealityInputPointerProfile'_ to make changes to it.
+    - At the top of the _'Pointer Settings'_, you should assign an invisible cursor prefab to the _'GazeCursor'_. If you downloaded the MRTK Examples folder, you can simply reference the included _'EyeGazeCursor'_ prefab.
+
+![MRTK](../Images/EyeTracking/mrtk_setup_eyes_gazesettings.jpg)
 
-![MRTK](../Images/EyeTracking/mrtk_setup_eyes_gazesettings.png)
 
 ### Accessing eye gaze data
-Now that your scene is set up to use Eye Tracking, let's take a look at how to access it in your scripts: 
-[Accessing Eye Tracking Data in your Unity Script](EyeTracking_EyeGazeProvider.md).
+Now that your scene is set up to use eye tracking, let's take a look at how to access it in your scripts: 
+[Accessing eye tracking data via EyeGazeProvider](EyeTracking_EyeGazeProvider.md) and [eye-supported target selections](EyeTracking_TargetSelection.md).
 
  
 ### Testing your Unity app on a HoloLens 2
-Building your app with Eye Tracking should be similar to how you would compile other HoloLens 2 MRTK apps. 
+Building your app with eye tracking should be similar to how you would compile other HoloLens 2 MRTK apps. 
 The only difference is that the *'Gaze Input'* capability is unfortunately not yet supported by Unity under 'Player Settings -> Publishing Settings -> Capabilities'. 
-To use Eye Tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your build Visual Studio project.
-Follow these steps:
+To use eye tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your built Visual Studio project.
 
+Follow these steps:
 1. Build your Unity project as you would normally do for _HoloLens 2_.
 2. Open your compiled Visual Studio project and then open the _'Package.appxmanifest'_ in your solution.
-3. Make sure to tick the _'Gaze Input'_ checkbox under _Capabilities_.
+3. Make sure to tick the _'GazeInput'_ checkbox under _Capabilities_.
 
 ![Enabling Gaze Input in Visual Studio](../Images/EyeTracking/mrtk_et_gazeinput.jpg)
 
-**IMPORTANT:** Don't forget to calibrate HoloLens for the current user. Without calibration, eye tracking won't work. Calibration can be found under system settings.  
+You don't see a _'GazeInput'_ capability? 
+   - Check that your system meets the [prerequisites for using MRTK](../GettingStartedWithTheMRTK.md) (in particular the Windows SDK version). 
+   - You can also manually add the entry by opening the appxmanifest in an XML editor and adding the following:
```

Comment:
I *think* this might be required after each Unity build - can you confirm/deny this is the case, and if it is the case (i.e. that this appx manifest change gets blown away on each build) can you add a BOLD note saying that this is required on each Unity build?

Reply:
This is only required if you build into a new folder. Otherwise, once you've set it once, you don't have to do it again. I can add a comment about that, but isn't it now outdated anyways, because it's set automatically?

Reply:
I added a note about it.